osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/config/AbstractOsgpSchedulerConfig.java

```diff
@@ -48,4 +70,52 @@ public OsgpScheduler osgpScheduler() throws SchedulerException {
         final Scheduler quartzScheduler = this.constructAndStartQuartzScheduler(schedulingConfigProperties);
         return new OsgpScheduler(quartzScheduler);
     }
+
+    private void testIfQuartzTablesExist() {
```

Comment:
Does this also work when startStopThreads = 1 (which is currently the case at Liander)?

Reply:
No, the component blocks the Apache Tomcat server from starting.